oughta.cabal

```diff
@@ -135,10 +135,10 @@ library
   build-depends:
     bytestring,
     containers,
-    exceptions >= 0.10 && < 0.11,
-    file-embed >= 0.0.16 && < 0.1,
-    hslua >= 2.3 && < 2.4,
-    text,
+    exceptions ^>= 0.10,
+    file-embed ^>= 0.0.16,
+    hslua ^>= 2.3,
+    text ^>= { 2, 2.1 },
```

Comment:
Similarly, I wonder if this should just be `>= 2 && < 2.2` (although that's about the same length as what you have here).